### PR TITLE
Adds descriptor to first line of signature

### PR DESCRIPTION
**Pull request by IAmHughes**
This is my first Sourcegraph contribution 🎉

A simple edit to add some context to the first line as it felt it was missing a descriptor, where the other lines have one.

### CODE REVIEW

**Review by emidoots — APPROVED** [review]
Nice, excited to have you on the team 🚀 Also stuff like this is super important, imagine the hundreds of people that will come after you and feel the same way :)

**Comment by IAmHughes** [comment]
> Nice, excited to have you on the team 🚀 Also stuff like this is super important, imagine the hundreds of people that will come after you and feel the same way :)

Thanks @slimsag - had you as reviewer as you were the suggestion from GH. Happy to 🚢 it when the final check passes. I haven't read into the `CONTRIBUTING.md` guidelines yet, is it fine to merge on my end since all checks passed? 👍

**Comment by emidoots** [comment]
Oh yeah, for sure, if you get an approval you're good to go in general. Sometimes people will leave suggestions and approve, or mention someone else who might want to review - in those cases you can just use your best judgement.

You can also enable auto merge as a sort of "I'm feeling lucky" if you feel confident about your changes & just want it to auto merge once one person approves.

**Pull request merged**
